src/function/signal/freqz.js

```diff
@@ -0,0 +1,108 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'freqz'
+
+const dependencies = [
+  'typed',
+  'add',
+  'multiply',
+  'Complex',
+  'divide',
+  'matrix'
+]
+
+export const createFreqz = /* #__PURE__ */ factory(name, dependencies, ({ typed, add, multiply, Complex, divide, matrix }) => {
+  /**
+     * Calculates the frequency response of a filter given its numerator and denominator coefficients.
+     *
+     * Syntax:
+     *    math.freqz(b, a)
+     *    math.freqz(b, a, w)
+     *
+     * Examples:
+     *   math.freqz([1, 2], [1, 2, 3], 4) // returns { h: [0.5+0j, 0.47685+0.2861j, 0.2500+0.75j,  -0.770+0.4625j], w: [0, 0.7853981633974483, 1.5707963267948966, 2.356194490192345 ] }
```

Comment:
Can you change the occurrences of `j` with `i`? To keep all documentation consistent. Also, there is a script that tests whether the examples work as advertised, and it doesn't understand `j` for complex numbers.